src/EditorFeatures/VisualBasicTest/SimplifyTypeNames/SimplifyTypeNamesTests.vb

```diff
@@ -977,7 +977,7 @@ End Namespace")
         Public Async Function TestSimplifyTypeInScriptCode() As Task
             Await TestAsync(
 "Imports System
-[|System.Console.WriteLine(0)|]",
+[|System.Console|].WriteLine(0)",
```

Comment:
old test span was incorrect (but the test still worked because of bugs in the analyzer).  With the PR we still offer the fix but on the right span.